### ISSUE TITLE
Mobile menu a11y fixes

### ISSUE THREAD

**Issue by joshkimux** [AIGAdc/dcdesignweek.org-11ty#150]
## Description

A couple of issues ranging from consider to should exist within the _beautiful_ mobile menu button. I'll break them down below and do my best to provide recommendations.

## "Close menu" for clarity (Consider ✔️ ) ✅

Currently, upon expanding the menu, the button label remains the same. Adjusting this to "Close menu" will provide a bit more clarity! This is a consider as it's more content based feedback.

<img width="460" alt="Screen Shot 2022-09-26 at 8 03 04 PM" src="https://user-images.githubusercontent.com/14154792/192401657-7361262b-d5d4-4bbc-91a0-ee5bf4416d1e.png">

## Menu items should not have `aria-label`s ✅

Given these are in the `nav` landmark in the header, we can assume folks will know these are pages they are navigating to. We can remove `aria-label`s to save up some space and practice the [first rule of aria (don't use aria unless we need to)](https://www.w3.org/TR/using-aria/#firstrule).

<img width="522" alt="Screen Shot 2022-09-26 at 8 22 03 PM" src="https://user-images.githubusercontent.com/14154792/192403474-e19f0d63-255c-45d7-b446-7579e5be02fb.png">

## Current page should have visual indicator (Should ⚠️ ) ✅

The mobile menu (and desktop menu) do not have visual indicators noting what page the user is on. We've talked about this one before, so just writing this up as a documented reminder 🎸 

<img width="368" alt="Screen Shot 2022-09-26 at 8 04 04 PM" src="https://user-images.githubusercontent.com/14154792/192401743-f74c1328-c5d2-478f-9657-4734bf0be9c7.png">

## Focus should be trapped (Should ⚠️ ) 🛑 (Removed mobile menu and dropdown for mobile menu and "jump to" dropdowns)

Currently, folks can tab right out of the menu and into the rest of the page, resulting in either:
- content being covered by the menu as the user scrolls down the page
- screen reader users accidentally triggering a tap on the open menu while swiping through the interface (this is hard to explain, but I can demo on my mobile device, we caught this several times during usability testing with screen reader users at the VA 😓 )

In other words, it should behave like a modal 😆 

https://user-images.githubusercontent.com/14154792/192402865-babeaa90-2ca3-48f7-b6ad-a37ca961e590.mov

<img width="377" alt="Screen Shot 2022-09-26 at 8 16 31 PM" src="https://user-images.githubusercontent.com/14154792/192402914-a36cc6a2-57e5-4ee5-8745-ae95120b4713.png">

## Home should exist in the mobile menu and maybe the desktop menu too (Should ⚠️ ) ✅

If we trap focus, things get a little complicated. Specifically, how might a user get to the homepage if it's not within the menu? Adding "Home" into the mobile menu can solve this and reduce the scope of what we wrap with the `nav` landmark.

## All wrapped up it might look like this

<img width="376" alt="Screen Shot 2022-09-26 at 8 26 23 PM" src="https://user-images.githubusercontent.com/14154792/192404052-b58af708-fafb-4b3d-9465-327bd897a105.png">
